test/docfx.Test/lib/ProcessUtilityTest.cs

```diff
@@ -25,7 +25,7 @@ public static class ProcessUtilityTest
 
                 File.Delete(fileName);
                 return Task.FromResult(0);
-            }, Path.GetFullPath($"{fileName}.lock"))));
+            }, $"{fileName}.lock")));
```

Comment:
What hell of indent!!

Reply:
the stylecop doesn't give me any error or warnings, so i believe it's valid?

---
In reply to: [183932244](https://github.com/dotnet/docfx/pull/2666#discussion_r183932244) [](ancestors = 183932244)

Reply:
stylecop is disable in test, too many false positives. you can reorder to the parameter to make it *look good*: `Lock(path, action)`